packages/styles/src/mixins/_notification.scss

```
@@ -72,7 +72,6 @@
 
   @include utilities-mx.high-contrast-mode {
     filter: none !important;
-    background-color: transparent !important;
     border: 2px solid WindowText;
```

```suggestion
    border: 2px solid CanvasText;
```
https://wiki.mozilla.org/Accessibility/Design_Guide ->
"Components not in the above categories should use CanvasText / Canvas (older implementations may use WindowText / Window)"

---

Good catch!